apis/config/v1beta1/configuration_types.go

```diff
@@ -169,6 +169,21 @@ type ControllerMetrics struct {
 	// metrics will be reported.
 	// +optional
 	EnableClusterQueueResources bool `json:"enableClusterQueueResources,omitempty"`
+
+	// LocalQueueMetrics configures whether metrics should be exposed per LocalQueue
```

Comment:
```suggestion
	// LocalQueueMetrics configures whether metrics should be exposed per LocalQueue
	// +optional
```

Could you add `+optional` marker to all optional fields?